lib/myfinance/resources/payable_account.rb

```diff
@@ -8,12 +8,19 @@ module Resources
     #
     class PayableAccount < FinancialAccount
 
+      def pay(id, entity_id, params = {})
+        http.put(endpoints(id, entity_id, :pay), body: { payable_account: params }) do |response|
+          respond_with_object(response, 'payable_account')
+        end
+      end
+
       private
 
-      def endpoints
+      def endpoints(id, entity_id, key)
         {
-          create: '/entities/:entity_id/payable_accounts'
-        }
+          create: '/entities/:entity_id/payable_accounts',
+          pay: '/entities/:entity_id/payable_accounts/:id/pay'
+        }[key].gsub(':entity_id', entity_id.to_s).gsub(':id', id.to_s)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.